src/sc_kpm/utils/action_utils.py

```diff
@@ -53,14 +52,14 @@ def execute_agent(
     arguments: Dict[ScAddr, IsDynamic],
     concepts: List[Idtf],
     initiation: Idtf = QuestionStatus.QUESTION_INITIATED.value,
-    reaction: QuestionStatus = QuestionStatus.QUESTION_FINISHED_UNSUCCESSFULLY,
+    reaction: QuestionStatus = QuestionStatus.QUESTION_FINISHED_SUCCESSFULLY,
     wait_time: int = COMMON_WAIT_TIME,
-) -> bool:
+) -> Tuple[ScAddr, bool]:
     keynodes = ScKeynodes()
     question = call_agent(arguments, concepts, initiation)
     wait_agent(wait_time, question, keynodes[QuestionStatus.QUESTION_FINISHED.value])
     result = check_edge(sc_types.EDGE_ACCESS_VAR_POS_PERM, keynodes[reaction.value], question)
-    return result
+    return question, result
```

Comment:
rename _question_ to _question_node_, better to represent the element type in the name of the variable